SOSS/extract/Simple_solver/simple_solver.py

```diff
@@ -17,11 +17,12 @@
 import warnings
 import emcee
 from scipy.ndimage.interpolation import rotate
-from SOSS.trace import get_uncontam_centroids as ctd
+from SOSS.dms import soss_centroids as ctd
 from SOSS.extract import soss_read_refs
 from SOSS.extract.simple_solver import plotting as plotting
 
 warnings.simplefilter(action='ignore', category=RuntimeWarning)
```

Comment:
I'd prefer warnings to be caught for each statement that cause them - they can be useful and should only be dismissed when you know for sure that this is okay to happen i.e.
```python
import warnings
with warnings.catch_warnings(record=True) as _:
    my_specific_piece_of_code_here.run()
```

Reply:
Address during review of simple solver.